modules/core/src/main/scala/scala/build/errors/FetchingDependenciesError.scala

```diff
@@ -4,9 +4,9 @@ import coursier.error.CoursierError
 
 import scala.build.Position
 
-final class FetchingDependenciesError(
+final case class FetchingDependenciesError(
```

Comment:
Same as above, this shouldn't be a case class.
```suggestion
final class FetchingDependenciesError(
```

Reply:
fixed